<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCaptureRegionDialog</class>
 <widget class="QDialog" name="SelectCaptureRegionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Capture Region</string>
  </property>
  <widget class="QPushButton" name="ClearMaskButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear Mask</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="my_qlabel" name="firstFrameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>731</width>
     <height>461</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
